.husky/pre-commit

```diff
@@ -5,7 +5,7 @@
 export FORCE_COLOR=1
 
 npm run lint
-npm run unit -- --onlyChanged
+npm test
```

Comment:
This should be `npm test -- --onlyChanged` so the pre-commit hook only runs the tests for files that changed. As we add more tests, running them all will take more time.

Reply:
```suggestion
npm test -- --onlyChanged
```